### ISSUE TITLE
Clean up the /etc/fstab on all nodes.

### ISSUE THREAD

**Issue by steven-basart** [centerforaisafety/cerberus-cluster#104]
There's a bunch of mounts still there from the nfs deployment that need to be removed.

**Comment by andriy-safe-ai** [comment]
Ran the following playbook to clean up the `/etc/fstab` on all node:
```
---

# cleanup /etc/fstab
- hosts: all
  gather_facts: true
  become: true
  tasks:
    - name: Unmount /mnt/fss/home
      mount:
        path: "/mnt/fss/home"
        state: unmounted
    - name: Unmount /mnt/fss/data
      mount:
        path: "/mnt/fss/data"
        state: unmounted
    - name: Remove the line from /etc/fstab
      lineinfile:
        path: /etc/fstab
        state: absent
        regexp: '^172\.16\.6\.6:/data\s+/mnt/fss/data\s+nfs\s+defaults\s+0\s+0$'
    - name: Remove the line from /etc/fstab
      lineinfile:
        path: /etc/fstab
        state: absent
        regexp: '^172\.16\.4\.200:/mnt/nfsshare/exports/data\s+/mnt/ha-nfs/data\s+nfs\s+defaults\s+0\s+0$'
    - name: Remove the line from /etc/fstab
      lineinfile:
        path: /etc/fstab
        state: absent
        regexp: '^172\.16\.6\.6:/home\s+/mnt/fss/home\s+nfs\s+defaults\s+0\s+0$'
    - name: Remove the line from /etc/fstab
      lineinfile:
        path: /etc/fstab
        state: absent
        regexp: '^172\.16\.4\.200:/mnt/nfsshare/exports/home\s+/mnt/ha-nfs/home\s+nfs\s+defaults\s+0\s+0$'
```

**Comment by steven-basart** [comment]
This isn't really fixed until we figure out #105
